tools/data-handler/test/test-data/valid/minimal/.cards/local/templates/test-template/template.json

```diff
@@ -1,4 +1,4 @@
 {
-  "buttonLabel": "Create a New Decision",
-  "namePrompt": "The title of the new decision:"
+    "buttonLabel": "",
```

Comment:
We just removed buttonLabel/namePrompt, so either this must be reverted and rebased or just remove them also from here

Reply:
I'll rebase this and fix if there are issues.